resources/core/charts/kubeless/charts/lambdas-ui/templates/configmap.yaml

```diff
@@ -21,10 +21,8 @@ data:
           {size:{ name: 'XL', memory: '1024Mi', cpu: '2.0', minReplicas: 10, maxReplicas: 20, description: '' }}
         ],
       subscriptionConfig:{
-        pushRequestTimeoutMS: 60000,
```

Comment:
To prevent, unnecessary debug hell, I propose this change to go with lambda-ui image bump PR.